salesperson_disable_mail_autosubscribe/models/sale_order.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _message_auto_subscribe_followers(self, updated_values, default_subtype_ids):
+        # By default, _message_auto_subscribe_followers adds the user_id of
+        # any record inheriting mail.thread to the list of followers.
+        # This removes this follower from the auto-subscribed followers.
+        res = super()._message_auto_subscribe_followers(
+            updated_values, default_subtype_ids
+        )
+        user_id = updated_values.get("user_id")
```

Comment:
Explain why you do it as post operation instead of removing the `user_id` from `updated_values` before calling `super` maybe?

Reply:
done